spec/author_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../author'
+
+describe Author do
+  before :each do
+    @author = Author.new('Clinton', 'Younge')
+  end
+
+  it 'should have a first name' do
+    expect(@author.first_name).to eq('Clinton')
+  end
+
+  it 'should have a last name' do
+    expect(@author.last_name).to eq('Younge')
+  end
```

Comment:
- [ ] As required for the project all public methods in all classes to have unit tests, kindly ensure that you add tests to the `#add_item()` method in the `Author` class